ingest/invoker.py

```diff
@@ -0,0 +1,31 @@
+from command import Command
+
+
```

Comment:
This class seems unnecessary -- if all it's doing is serving as a list of Commands, it's just a wrapper around list, and so the code would be clearer if the classes that were using this just kept their own list and used map.  Is there an intent to put centralized logging/error handling logic here later?

Reply:
e405fb2